app/routes/index.js

```diff
@@ -3,14 +3,22 @@ import SessionRoute from "./session";
 import AjaxPromise from "stock/utils/ajax-promise";
 
 export default SessionRoute.extend({
-  model() {
+  async model() {
     let canViewDashboard = this.get("session.currentUser.canManageOrders");
-    var recentlyUsedDesignations = this.get("store").query("designation", {
-      recently_used: true,
-      shallow: true
+    const recentlyUsedDesignations = await this.get("store").query(
```

Comment:
@abulsayyad123 Candidate for your new recently_used method here

Reply:
Yes. I have just merged `recent_used` on `package_type`. Once, it's approved by QA's we apply it to other models as well. For now, it is good to go.